### DISCUSSION TITLE
Kong v2.8.2 missing on Docker Hub

### DISCUSSION THREAD

**Discussion by vinayakkulkarni** [Kong/kong#9638] [Help] [answered]
There's a release https://github.com/Kong/kong/releases/tag/2.8.2 but no Docker image on Docker Hub  https://hub.docker.com/_/kong/tags

Any process for adding the 2.8.2 release on the registry?

**Comment by hbagdi** [comment]
cc @hutchic @VicYP

**Comment by hutchic** [comment, marked as answer]
Sorry about that.

We've opted to not release 2.8.2 and to instead skip to 2.8.3 which is now [available](https://hub.docker.com/_/kong/tags)
